app/src/main/java/com/quran/labs/androidquran/ui/fragment/QuranSettingsFragment.kt

```diff
@@ -88,6 +103,19 @@ class QuranSettingsFragment : PreferenceFragment(),
       return true
     }
 
-    return super.onPreferenceTreeClick(preferenceScreen, preference)
+    return super.onPreferenceTreeClick(preference)
+  }
+
+  private fun View.setZeroPaddingToLayoutChildren() {
```

Comment:
why is this necessary? it's also there in `AboutFragment` - should we move this somewhere and share it instead?